script/Deploy.s.sol

```diff
@@ -2,18 +2,34 @@
 
 pragma solidity 0.8.23;
 
-import { DeployBase } from "./DeployBase.s.sol";
+import { Script, console2 } from "../lib/forge-std/src/Script.sol";
 
-contract Deploy is DeployBase {
+import { IMinterGateway } from "../src/interfaces/IMinterGateway.sol";
+
+import { DeployBase } from "./DeployBase.sol";
+
+contract Deploy is Script, DeployBase {
     // NOTE: Ensure this is the correct TTG Registrar testnet/mainnet address.
     address internal constant _TTG_REGISTRAR = 0x1EFeA064121f17379b267db17aCe135475514f8D;
```

Comment:
I know this would coupled Protocol and TTG but shouldn't we use `ContractHelper.getContractFrom()` to retrieve the address of the Registrar?
Cause here if we forget to update this address, we will have to re-deploy and we may only notice it later.

Reply:
This deploy script is only to deploy protocol alone, outside the context of ttg. There is no way for this script to know what the TTG address will be, since that is TTG-related procedures, and TTG can be deployed before or after protocol. Full deployments will happen from `m-core`.